src/components/table/table.tsx

```diff
@@ -153,9 +154,32 @@ function EnhancedTableToolbar({
             }}
           >
             <h3 className={$table.tableTitle}>{type}</h3>
-            <button onClick={onOpenInput} className={`${$button.default}`}>
-              Add {type}
-            </button>
+
+            <div
+              style={{
+                display: "flex",
+                justifyContent: "space-around",
+                alignItems: "center",
+              }}
+            >
+              <SearchBar
+                id="search-input"
+                placeholder="Search"
```

Comment:
I think 'Filter' is more applicable to this UX. I know the figma design calls for 'Search', but this is filtering a list, not doing a traditional search, which would return a new list of search results. Just a comment, no big deal.

Reply:
I didn't build the search functionality, just the UI component. Do you think it would benefit us more to build a traditional search rather than the filtering?

Reply:
No, I was just saying we should change the placeholder text from 'Search' to 'Filter', but I'm aware that you didn't create that placeholder value in this PR.

Reply:
Do you think my listing error has to do with my version of node I’m using? It keeps saying prettier is not run, but I’ve even run it separate from the push check and says it’s ok